.github/actions/reusable-python-build_poetry/action.yml

```diff
@@ -0,0 +1,62 @@
+name: Build poetry package
+description: Build a poetry package with a meta.yaml file
+
+inputs:
+    python-version:
+        description: 'Python version to use (eg: 3.10)'
+        required: true
+        type: string
+    extra-repository:
+        description: 'Boolean to indicate if extra repository is needed'
+        required: false
+        type: boolean
+        default: false
+    repo-user:
+        description: 'Repository username'
+        required: false
+        type: string
+    repo-token:
+        description: 'Repository token'
+        required: false
+        type: string
+
+outputs:
+    dir-path:
+        description: 'Path to the directory containing the built package'
+        value: ${{ steps.define-path.outputs.dir-path }}
+
+runs:
+    using: "composite"
+    steps:
+        -   name: Set up Python version
+            uses: actions/setup-python@v5
+            with:
+                python-version: ${{ inputs.python-version }}
+        -   name: Setup poetry env
+            uses: MiraGeoscience/CI-tools/.github/actions/reusable-python-setup_poetry@main
+            with:
+                cache-number: 20
+                runner-os: ubuntu-latest
+                extra-repository: ${{ inputs.extra-repository }}
+                repo-user: ${{ inputs.repo-user }}
+                repo-token: ${{ inputs.repo-token }}
+
+        -   name: Build package
+            id: build_package
+            run: |
+                    poetry build
+            shell: bash
+        -   name: Define path
+            id: define-path
+            run: |
+                DIR_PATH=$(echo "dist")
+                echo "dir-path: ${DIR_PATH}"
+                echo "dir-path=$( echo $DIR_PATH)" >> $GITHUB_OUTPUT
+                echo "dir-path=$( echo $DIR_PATH)" >> $GITHUB_ENV
```

Comment:
**question:** is necessary to expose both as an env variable and in the output?

Reply:
No, it wasn't. I corrected it.